<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseBuiltinPaletteDialog</class>
 <widget class="QDialog" name="ChooseBuiltinPaletteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Built-in Palette</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qt-resources/qt_widgets.qrc">
    <normaloff>:/GPlates-icon-160x160-opaque.png</normaloff>:/GPlates-icon-160x160-opaque.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="colorbrewer_groupbox">
     <property name="title">
      <string>ColorBrewer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="colorbrewer_sequential_groupbox">
        <property name="title">
         <string>Sequential</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="multi_hue_groupbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>6</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Multi-hue</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item row="1" column="5">
               <widget class="QWidget" name="YlOrRd_placeholder" native="true"/>
              </item>
              <item row="1" column="4">
               <widget class="QWidget" name="YlOrBr_placeholder" native="true"/>
              </item>
              <item row="0" column="2">
               <widget class="QWidget" name="GnBu_placeholder" native="true"/>
              </item>
              <item row="0" column="3">
               <widget class="QWidget" name="OrRd_placeholder" native="true"/>
              </item>
              <item row="0" column="4">
               <widget class="QWidget" name="PuBu_placeholder" native="true"/>
              </item>
              <item row="0" column="5">
               <widget class="QWidget" name="PuBuGn_placeholder" native="true"/>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="BuPu_placeholder" native="true"/>
              </item>
              <item row="0" column="0">
               <widget class="QWidget" name="BuGn_placeholder" native="true"/>
              </item>
              <item row="1" column="1">
               <widget class="QWidget" name="RdPu_placeholder" native="true"/>
              </item>
              <item row="1" column="2">
               <widget class="QWidget" name="YlGn_placeholder" native="true"/>
              </item>
              <item row="1" column="3">
               <widget class="QWidget" name="YlGnBu_placeholder" native="true"/>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="PuRd_placeholder" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="single_hue_groupbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>3</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Single hue</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item row="1" column="1">
               <widget class="QWidget" name="Purples_placeholder" native="true"/>
              </item>
              <item row="1" column="2">
               <widget class="QWidget" name="Reds_placeholder" native="true"/>
              </item>
              <item row="1" column="0">
               <widget class="QWidget" name="Oranges_placeholder" native="true"/>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="Greens_placeholder" native="true"/>
              </item>
              <item row="0" column="2">
               <widget class="QWidget" name="Greys_placeholder" native="true"/>
              </item>
              <item row="0" column="0">
               <widget class="QWidget" name="Blues_placeholder" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="colorbrewer_diverging_classes_label">
             <property name="text">
              <string>Classes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="colorbrewer_sequential_classes_spinbox">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>3</number>
             </property>
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>9</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="colorbrewer_diverging_groupbox">
        <property name="title">
         <string>Diverging</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QWidget" name="BrBG_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="PiYG_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="PRGn_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="PuOr_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="RdBu_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="RdGy_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="RdYlBu_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="RdYlGn_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="Spectral_placeholder" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="colorbrewer_sequential_classes_label">
             <property name="text">
              <string>Classes:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="colorbrewer_diverging_classes_spinbox">
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="minimum">
              <number>3</number>
             </property>
             <property name="maximum">
              <number>11</number>
             </property>
             <property name="value">
              <number>11</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="colorbrewer_discrete_invert_layout">
        <item>
         <widget class="QCheckBox" name="colorbrewer_discrete_checkbox">
          <property name="text">
           <string>Discrete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="colorbrewer_invert_checkbox">
          <property name="text">
           <string>Invert</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SCM_groupbox">
     <property name="title">
      <string>Scientific Colour Maps</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="SCM_sequential_groupbox">
        <property name="title">
         <string>Sequential</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QWidget" name="SCM_batlow_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_hawaii_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_oslo_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_lapaz_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_lajolla_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_buda_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_davos_placeholder" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="SCM_tokyo_placeholder" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="SCM_diverging_groupbox">
          <property name="title">
           <string>Diverging</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QWidget" name="SCM_vik_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_roma_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_broc_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_berlin_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_lisbon_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_bam_placeholder" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SCM_multi_sequential_groupbox">
          <property name="title">
           <string>Multi-sequential</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QWidget" name="SCM_oleron_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="SCM_bukavu_placeholder" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="other_groupbox">
     <property name="title">
      <string>Other</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="age_groupbox">
          <property name="title">
           <string>Age</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="age_legacy_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="age_traditional_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="age_modern_placeholder" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="topo_groupbox">
          <property name="title">
           <string>Topography</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QWidget" name="topo_etopo1_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="topo_geo_placeholder" native="true"/>
           </item>
           <item>
            <widget class="QWidget" name="topo_relief_placeholder" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="main_buttonbox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>main_buttonbox</sender>
   <signal>accepted()</signal>
   <receiver>ChooseBuiltinPaletteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_buttonbox</sender>
   <signal>rejected()</signal>
   <receiver>ChooseBuiltinPaletteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
